stdlib/2and3/sqlite3/dbapi2.pyi

```diff
@@ -15,9 +15,9 @@ Date = date
 Time = time
 Timestamp = datetime
 
-def DateFromTicks(ticks): ...
-def TimeFromTicks(ticks): ...
-def TimestampFromTicks(ticks): ...
+def DateFromTicks(ticks: float) -> datetime: ...
```

Comment:
Actually returns `sqlite3.dbapi2.Date`, which is `datetime.date`, not `datetime.datetime`

Reply:
Good catch. I've incorporated this change and your other recommendations.